tests/common/graph/checks/test_policy_packaging.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+
+def test_graph_policies_packaging_requirements() -> None:
+    """Checks, if all graph_checks folder have a __init__.py file"""
```

Comment:
Could you indicate here also why is that important? Just for future developers :)

Reply:
done 🙂